src/main/cc/wfa/measurement/common/macros.h

```diff
@@ -15,16 +15,44 @@
 #ifndef WFA_MEASUREMENT_COMMON_MACROS_H_
 #define WFA_MEASUREMENT_COMMON_MACROS_H_
 
+#ifndef RETURN_IF_ERROR
```

Comment:
why do you want to define a local copy here?
We are using the same macros from the PJC repo.

Reply:
Other C++ code shouldn't need a dependency on PJ&C for these macros. (And this code, along with other common code, should eventually be pulled to its own repo with minimal deps.)